### PR TITLE
Openg2p Tasks

### PR DESCRIPTION
**Pull request by rrkas**
- task models
- task UI

### CODE REVIEW

File: openg2p/models/beneficiary_to_batch_wizard.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def create_batch(self):
                     {
                         "name": self.batch_name
                         + "-"
-                        + str(datetime.now().strftime("%d%m%y-%I:%M")),
+                        + str(datetime.now().strftime("d-%m-%Y-%H:%M")),
```
Comment (SubhamPramanik) [inline openg2p/models/beneficiary_to_batch_wizard.py:78]: Wasn't this date time change made in the previous PR?

File: openg2p/models/openg2p_beneficiary.py [hunk 2]
```diff
@@ -585,6 +645,7 @@ def _search_age(self, operator, val):
         res = []
         bs = self.env["openg2p.beneficiary"].search([])
         for b in bs:
+            print(b.age, operator, val)
```
Comment (SubhamPramanik) [inline openg2p/models/openg2p_beneficiary.py:648]: Remove print statement

File: openg2p/models/openg2p_beneficiary_api.py [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+            for category in category_list:
+                res.append(category.api_json())
+            return {
+                "status": 200,
+                "message": "Success",
+                "count": len(res),
+                "categories": res,
+            }
+        except BaseException as e:
+            return {"status": 200, "error": str(e)}
```
Comment (SubhamPramanik) [inline openg2p/models/openg2p_beneficiary_api.py:23]: Status shouldn't be 200 in case of error

File: openg2p/models/openg2p_beneficiary_api.py [hunk 4]
```diff
@@ -0,0 +1,265 @@
+from odoo import fields
```
Comment (SubhamPramanik) [inline openg2p/models/openg2p_beneficiary_api.py:1]: Weren't all these Program and Beneficiary APIs already created in earlier PRs?

File: openg2p_disbursement_advice/models/openg2p_disbursement_api.py [hunk 5]
```diff
@@ -0,0 +1,105 @@
+from odoo.http import Controller, route, request
```
Comment (SubhamPramanik) [inline openg2p_disbursement_advice/models/openg2p_disbursement_api.py:1]: Same over here. I think these were previously merged. Could you confirm once? 

https://github.com/OpenG2P/openg2p-erp/pull/97

**Review by SubhamPramanik — CHANGES_REQUESTED** [review]
Please check if you have changes that wasn't pulled from master

File: openg2p/models/beneficiary_to_batch_wizard.py (hunk 1 above)
Comment (rrkas) [reply openg2p/models/beneficiary_to_batch_wizard.py:78]: error in date format
d -> %d 
will be pushed in next commit

File: openg2p/models/openg2p_beneficiary_api.py (hunk 4 above)
Comment (rrkas) [reply openg2p/models/openg2p_beneficiary_api.py:1]: They have been.
They are still visible in master.
And still these are shown as changes.

File: openg2p/models/openg2p_beneficiary_api.py (hunk 3 above)
Comment (rrkas) [reply openg2p/models/openg2p_beneficiary_api.py:23]: Changed status to 500

**Review by SubhamPramanik — APPROVED** [review]

**Pull request merged**